kafka/endring-paa-oppf-bruker-ingestor/src/test/kotlin/no/nav/amt/tiltak/ingestors/endring_paa_oppf_bruker_ingestor/EndringPaaBrukerIngestorImplIntegrationTest.kt

```diff
@@ -32,7 +32,7 @@ class EndringPaaBrukerIngestorImplIntegrationTest {
 
 	lateinit var endringPaaBrukerIngestorImpl: EndringPaaBrukerIngestor
 	lateinit var deltakerService: DeltakerService
-	lateinit var brukerService: BrukerService
+	lateinit var brukerService: BrukerServiceImpl
```

Comment:
Litt usikker på hvorfor denne endringen men er sikkert ok ..?

Reply:
grunnen er at vi trenger en port i core for å få tilgang til brukerService fra andre pakker siden leesha readeren ligger i en annen pakke. Kunne gått gjennom deltakerservice, men følte det ble rotete